docs/users/SystemRequirements.md

```diff
@@ -0,0 +1,43 @@
+# System Requirements
+!!! note "This page does not include the requirements to run any supported games. Please refer to the game store page for details."
+
+This page lists the minimum and recommend system requirements to install and run the app on all supported platforms. 
+
+## Windows 
+
+|| Minimum | Recommended |
+| --- | --- | --- |
+| OS | Windows 10 or higher (64-bit OS required) | Windows 10/11 with updates |
+| Processor | Intel®️ Core™️ i5-7500 | Intel®️ Core™️ i7-7500 |
+| Memory | 4 GB RAM | 16 GB RAM |
+| Graphics | 2GB VRAM | 4GB VRAM |
+| Storage | 10GB available space* | 200GB available space* |
```

Comment:
Where did these values come from?

Reply:
These "requirements" don't really make a lot of sense. The only thing we require on Windows is Windows 10 or newer. We don't care what hardware the PC has.

Reply:
We should either not have the requirements or track down actual values. I'm more in favor of the latter. I think we can remove all of the windows requirements except for Windows 10/11 for now, as that's about all we currently know.

Reply:
fixed